ethics/20-tiktok-fr.md

```diff
@@ -0,0 +1,460 @@
+des recommandations faites à d'autres comptes TikTok,
+peut-être jugés plus enclins à la radicalisation,
+ou plus facilement victimes de distraction stratégique -
+je pense notamment aux chercheurs en IA plus prompts 
+à s'émerveiller devant les gadgets qui génèrent des images photoréalistes
+qu'à s'intéresser aux dangers des algorithmes de recommandation.
+
+En fait, le problème de l'audit est plus complexe encore,
+car les recommandations faites par TikTok ne sont pas nécessairement représentatives
+des recommandations que fera TikTok dans les années à venir.
+Après tout, TikTok est encore dans une phase d'énorme croissance.
+Dans ce contexte, les autorités chinoises ont en fait tout intérêt à persévérer
+dans la quête de parts de marché,
+de sorte que sa télécommande fonctionne sur davantage de téléphones à travers le monde.
+Mais pour y arriver, TikTok a ainsi tout intérêt à éviter les controverses,
+comme des recommandations TikTok douteuses qui soulèveraient des préoccupations.
+Dès lors, il n'est pas étonnant que, pour l'instant,
+la télécommande de TikTok soit encore principalement utilisée pour diffuser des contenus distrayants.
+
+En fait, la principale préoccupation à avoir vis-à-vis de TikTok ne devrait pas être
+les recommandations faites aujourd'hui par l'algorithme de TikTok.
+La principale préoccupation est davantage celle de la gouvernance de cette télécommande de TikTok.
+Qui, aujourd'hui et demain, aura le pouvoir de reprogrammer cette télécommande ?
+À qui sommes-nous en train de céder le pouvoir de la gestion du flux de l'information à travers le monde ?
+
+Plus que comme une attaque frontale sur la démocratie,
+TikTok ressemble à un cheval de Troie, une offrande distrayante
+qui pourrait un jour déclencher l'effondrement des démocraties.
+
+
+## Le problème de la gouvernance algorithmique
+
+Alors, cette vidéo se concentre sur TikTok,
+parce que c'est un exemple particulièrement frappant,
+mais les préoccupations que nous avons soulevées s'appliquent en fait 
+pour toutes les commandes les plus puissantes du monde.
+Qui contrôle les télécommandes de YouTube, de Facebook, d'Instagram, de Twitter et de LinkedIn ?
+Qui les contrôleront demain ?
+Et qui contrôleront ceux qui ont le contrôle de ces télécommandes ?
+
+Vous le savez certainement, Twitter a récemment été racheté par Elon Musk,
+un milliardaire qui n'a pas tardé pour prendre des mesures qui inquiètent fortement,
+tout ceux qui s'intéressent à la qualité de l'information,
+au cyber-harcement et à la prolifération dangereuse des discours de haine.
+Twitter appartient par ailleurs en partie à des fonds liés à la monarchie de l'Arabie Saoudite,
+connue pour ses abus récurrents des droits humains,
+et a déjà eu plusieurs employés inculpés pour espionnage au profit de ce pays.  
+https://www.theguardian.com/technology/2022/oct/28/twitter-takeover-fears-raised-over-disinformation-and-hate-speech
```

Comment:
et aussi le Quatar https://www.arabianbusiness.com/industries/technology/saudis-prince-alwaleed-qatar-wealth-fund-meet-the-investors-who-funded-elon-musks-twitter-buyout